### PR TITLE
[skip vbump]  Setting to version 0.1.2.9019

### PR DESCRIPTION
**Pull request by arkadiuszbeer**

### CODE REVIEW

File: NEWS.md [hunk 1]
```diff
@@ -1,3 +1,7 @@
+# rlistings 0.1.2.9019
+  
+  ### Enhancements
+
 # rlistings 0.1.1.9019
```
Comment (Melkiades) [inline NEWS.md:5]: Should the other go away? like put 1.2.0 right?

**Review by Melkiades — CHANGES_REQUESTED** [review]

**Review by Melkiades — APPROVED** [review]
Lgtm anyway! thanks @arkadiuszbeer

**Comment by pawelru** [comment]
This is not that easy as just put higher version and that's it. We need to reconcile the scope that has been released already and the scope that came afterwards and amend news file accordingly. Let's do that correctly other than just fast

**Review by shajoezhu — APPROVED** [review]

**Comment by Melkiades** [comment]
> This is not that easy as just put higher version and that's it. We need to reconcile the scope that has been released already and the scope that came afterwards and amend news file accordingly. Also, usually the version cutoff is followed by tagging on GH so please do this as well.
> 
> Let's do that correctly other than just fast

Right, I will do it!

File: DESCRIPTION [hunk 2]
```diff
@@ -1,6 +1,6 @@
 Package: rlistings
 Title: Clinical Trial Style Data Readout Listings
-Version: 0.1.1.9019
+Version: 0.1.2.9001
```
Comment (shajoezhu) [inline DESCRIPTION:3]:
```suggestion
Version: 0.1.3.9019
```

File: NEWS.md [hunk 3]
```diff
@@ -1,4 +1,8 @@
-# rlistings 0.1.1.9019
+# rlistings 0.1.2.9001
```
Comment (shajoezhu) [inline NEWS.md]:
```suggestion
# rlistings  0.1.3.9019
```

**Review by shajoezhu — APPROVED** [review]

File: NEWS.md [hunk 4]
```diff
@@ -1,6 +1,13 @@
-# rlistings 0.1.1.9019
+## rlistings 0.1.3.9019
```
Comment (shajoezhu) [inline NEWS.md:1]:
```suggestion
## rlistings 0.1.2.9019
```

File: NEWS.md [hunk 5]
```diff
@@ -1,6 +1,13 @@
...
+
+### Enhancements
  * Extend page-size machinery in pagination by allowing the page specification (page_type, pg_width,
    pg_height, font_family, font_size) to be transformed into lpp (lines per page) and cpp (characters per page).
+ * New function `export_as_txt` to support output saved in plain text.
+
```
Comment (shajoezhu) [inline NEWS.md:7]:
```suggestion
```

File: NEWS.md [hunk 6]
```diff
@@ -1,6 +1,13 @@
-# rlistings 0.1.1.9019
+## rlistings 0.1.3.9019
+
+### Enhancements
  * Extend page-size machinery in pagination by allowing the page specification (page_type, pg_width,
    pg_height, font_family, font_size) to be transformed into lpp (lines per page) and cpp (characters per page).
+ * New function `export_as_txt` to support output saved in plain text.
+ 
+## rlistings 0.1.2
```
Comment (shajoezhu) [inline NEWS.md:8]:
```suggestion
```

File: NEWS.md [hunk 7]
```diff
@@ -1,6 +1,13 @@
...
+
+### Enhancements
```
Comment (shajoezhu) [inline NEWS.md:10]:
```suggestion

### Enhancements
```

File: NEWS.md [hunk 8]
```diff
@@ -1,6 +1,11 @@
-# rlistings 0.1.1.9019
+## rlistings 0.1.2.9019
+
+### Enhancements
  * Extend page-size machinery in pagination by allowing the page specification (page_type, pg_width,
    pg_height, font_family, font_size) to be transformed into lpp (lines per page) and cpp (characters per page).
+ * New function `export_as_txt` to support output saved in plain text.
+
+### Enhancements
```
Comment (shajoezhu) [inline NEWS.md:8]:
```suggestion
```

File: NEWS.md [hunk 9]
```diff
@@ -1,6 +1,10 @@
-# rlistings 0.1.1.9019
+## rlistings 0.1.2.9019
+
+### Enhancements
  * Extend page-size machinery in pagination by allowing the page specification (page_type, pg_width,
    pg_height, font_family, font_size) to be transformed into lpp (lines per page) and cpp (characters per page).
+ * New function `export_as_txt` to support output saved in plain text.
+
```
Comment (shajoezhu) [inline NEWS.md:7]:
```suggestion
```

**Review by shajoezhu — APPROVED** [review]
lgtm!

**Pull request merged**